va/va.go

```diff
@@ -738,6 +832,12 @@ func (va *ValidationAuthorityImpl) PerformValidation(ctx context.Context, domain
 	}
 	vStart := va.clk.Now()
 
+	var remoteError chan *probs.ProblemDetails
```

Comment:
Talked offline, and roland convinced me this is okay for now. One thing that came out of the conversation: We should treat `ProblemDetails` as a separate thing from `error` even though it has a similar property (that lots of functions early-return with a `ProblemDetails`). That would avoid some of the boxing errors we see here.